src/camera.py

```diff
@@ -0,0 +1,111 @@
+from os import getcwd
+
+from picar.SunFounder_PCA9685 import Servo
+from picar import filedb
+
+
+'''
+Camera class to control panning and tilting.
+
+Creates camera object that can be operated on using pan() and tilt().
+pan() operates [0,180] degrees. tilt() operates [0,150] degrees. These 
+functions can be calibrated using calibrate_pan() and calibrate_tilt().
+Calibration adds values to the pan() or tilt().
+'''
+class Camera:
+    PAN_CHANNEL = 1
+    TILT_CHANNEL = 2
+    MAX_PAN_DEGREE = 180
+    MAX_TILT_DEGREE = 150
+    MIN_DEGREE = 0
+    MAX_CALI_DEGREE = 180
+    MIN_CALI_DEGREE = -180
```

Comment:
what

Reply:
If you want to refer to 0 as 180 and work in 180 to 360, why not give that possibility? The way the validation works it won't break the servo, and otherwise I don't know what ranged to put for the calibration, it would all be arbitrary.

Reply:
I don't understand why we'd be working in 180 to 360 though?

Reply:
Me neither. But I had the option to calibrate between the range -50, 50 which seemed to be a decent middle ground. Would that be a better alternative?

Reply:
I mean, would it be so bad if we just didn't do any calibration? Clearly no one complained about their calibration functions not working.

Reply:
because it was using the values in the offset config file. You showed it a while ago, it was actually grabbing the offset values, just not rewriting them if you wanted to change them.

Reply:
Right, but those values are the default values for a reason. What do we gain by changing them?

Reply:
I believe having some flexibility and extra functionality on the camera would not be a hindrance. But as team lead and technical lead it is your decision.